query.go

```diff
@@ -475,6 +484,24 @@ func getAudioFilter(label *string) driver.FilterFn {
 	return filter
 }
 
+func SelectDriverByDeviceId(uuid string) driver.Driver {
+	filter := func(driver driver.Driver) bool {
+		for _, prop := range driver.Properties() {
+			if prop.DeviceID == uuid {
```

Comment:
So two things: this UUID isn't very stable and can change if discovery changes, so I'm not sure this is the best way to get a device. The other (unless I'm missing something) is that the state of the driver has nothing to do with the OS state of the driver. How is the state being updated. It seems like overall what this code in conjunction with the RDK is doing is just checking if the file still exists (status is unused). In that case, is there any gostream change necessary?